### DISCUSSION TITLE
Need help with Wifi adapters

### DISCUSSION THREAD

**Discussion by Mausbiber** [nzymedefense/nzyme#574] [Questions]
Hello,
I need your help with 3 x questions:
1. is it possible to monitor both wifi channels (2.4 / 5 GHz) simultaneously with only one wifi adapter?
2. if this is not possible, how do I set the Wifi adapters so that one is listening on 2.4 GHz and one on 5 GHz?
3. and the most important question, which Wifi adapter can you recommend to me that a) works out-of-the-box with a Raspberry Pi, b) works with both 2.4 GHz and 5 GHz and c) is currently still available for purchase?

**Comment by lennartkoopmann** [comment]
Hi,

> 1. is it possible to monitor both wifi channels (2.4 / 5 GHz) simultaneously with only one wifi adapter?

Yes, you can combine channels as you wish. See: https://www.nzyme.org/docs/installation-configuration/configuration-reference#802_11_monitorschannels

> 3. and the most important question, which Wifi adapter can you recommend to me that a) works out-of-the-box with a Raspberry Pi, b) works with both 2.4 GHz and 5 GHz and c) is currently still available for purchase?

The recommended and tested WiFi adapters are listed on the requirements page: https://www.nzyme.org/docs/installation-configuration/requirements/#required-one-or-more-wifi-adapters-that-support-monitor-mode-on-your-operating-system
